src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -441,15 +410,18 @@
   "Updates": "Updates",
   "UpdateScriptPathHelpText": "Path to a custom script that takes an extracted update package and handle the remainder of the update process",
   "Uptime": "Uptime",
+  "Url": "'Url'",
```

Comment:
Why is this in single quotes. I think it was just like that in the original code so it wouldn't interfere with the variable named the same.